pkg/proxy/backend/backend_conn_mgr.go

```diff
@@ -640,6 +669,9 @@ func (mgr *BackendConnManager) checkBackendActive() {
 		return
 	}
 	backendIO := mgr.backendIO.Load()
+	if backendIO == nil && mgr.lastPauseTime != 0 && monotime.Since(mgr.lastPauseTime) < mgr.config.PausedConnTimeout {
```

Comment:
Why use `&&`?
If `backendIO==nil` and the other conditions are unsatisfied, the next line will panic.

Reply:
Yes, I have found it during test. It has been fixed.